Tests/PerformanceTests.swift

```diff
@@ -0,0 +1,124 @@
+// Created by Cal Stephens on 2/14/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import Foundation
+import XCTest
+
+@testable import Lottie
+
+// MARK: - PerformanceTests
+
+final class PerformanceTests: XCTestCase {
```

Comment:
Good idea, I never tried the `measure` API because it doesn't return values that you could compare. It seems like a good idea to have both so you can use the `measure` ones locally, will take a look.